Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -132,10 +132,10 @@ private string ConvertJToken(JToken jToken)
                         _ => value?.ToString()?.Replace(',', '.'), // Replace decimal commas.
                     };
                 case JArray jArray:
-                    return $"new[] {{ {string.Join(", ", jArray.Select(ConvertJToken))} }}";
+                    return $"new[] \n\t\t\t{{ {string.Join(", ", jArray.Select(ConvertJToken))} \n\t\t\t}}";
                 case JObject jObject:
                     // Using a quoted string so it doesn't mess up the syntax highlighting of the rest of the code.
-                    return T["\"FIX ME! Couldn't determine the actual type to instantiate.\" {0}", jObject.ToString()];
+                    return $"\n\t\t\t\tnew ListValueOption {{ Name = \"{jObject["name"]}\", Value = \"{jObject["value"]}\" }}";

```

Comment:
This would tightly couple this code to the ContentFields module. It's OK-ish to add special cases for modules but let's also keep the option open to handle if we encounter something unknown, as it is with the "FIX ME!" message. Also, the comment above it is not current anymore.

Reply:
Hmm so maybe check to see if `jObject` has both a `name` and `value` property? Will remove the comment.

Reply:
Added a check for `name` and `value` properties. If either does not exist then return `FIX ME!`.

Reply:
That's good, thank you. Just to confirm, there's no other, more direct indication in the `JObject` or its parent object that we have a `ListValueOption` at hand, right?

Reply:
Not that I am aware of.